build.sh

```diff
@@ -176,9 +173,8 @@ patch_osbuild() {
     mv /usr/bin/osbuild-mpp /usr/lib/osbuild/tools/
 
     # Now all the software is under the /usr/lib/osbuild dir and we can patch
-    cat /usr/lib/coreos-assembler/0004-fscache-add-eviction-log-statement.patch                   \
-        /usr/lib/coreos-assembler/0001-stages-qemu-sanity-check-created-image.patch               \
-            | patch -d /usr/lib/osbuild -p1
+   #cat your.patch \
```

Comment:
I think we should keep a comment here that says to freeze osbuild if we ever need to add patches here again. Or alternatively, delete all of `patch_osbuild()`.

Reverting the commit to add patches will also revert the freeze that can then be updated.

Reply:
depending on the situation I don't think it's required that we pin OSBuild if we patch.

If it's an area of code that changes a lot then it might be worth it to pin, if not then maybe not.

I'm OK with deleting the whole function, but thought for now it might be worth keeping the scaffolding in place.

Let me know what you think is best.

Reply:
Leaning towards nuking it for now with the understanding that it's only a `git revert` away if we ever need to carry patches again in the future.

Reply:
sorry I missed this in the final upload

followup in https://github.com/coreos/coreos-assembler/pull/3809